README.md

```diff
@@ -0,0 +1,105 @@
+# Audio Waveform Image Generator
+
+This repository only contains the homebrew formula to easily install `audiowaveform` via homebrew. For other operating systems and code [see the original repository](https://github.com/bbcrd/homebrew-audiowaveform).
+
+[![Build Status](https://travis-ci.org/bbcrd/homebrew-audiowaveform.png?branch=master)](https://travis-ci.org/bbcrd/audiowaveform)
+
+**audiowaveform** is a C++ command-line application that generates waveform data
```

Comment:
To avoid duplication I'd prefer to keep all the usage info in the original project's ReadMe. The ReadMe here should only describe Homebrew specifics.